src/sidebars/sidebar-controller.js

```diff
@@ -51,6 +51,10 @@
                 this._showFavourites = true;
             }
         }
+
+        openGithub() {
+            fin.desktop.System.openUrlWithBrowser('https://github.com/ScottLogic/bitflux-openfin');
+        }
```

Comment:
Looks good, but can we run the `fin` API call through the current window service? Should be safer by using `ready` too.